### PR TITLE
RELATED: RAIL-3622 changes to parentFilterOverAttribute

### PR DESCRIPTION
**Pull request by no23reason**
Related PR in SDK: https://github.com/gooddata/gooddata-ui-sdk/pull/1679

Also some remnant from the recent project -> workspace move
in auto-generated file was fixed (here project is correct, not workspace).

### CODE REVIEW

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-ui-documentation-validation/836/ : SUCCESS in 1m 35s
- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-oss-check/549/ : SUCCESS in 9s

File: docs/10_vis__attribute_filter_button_component.md [hunk 1]
```diff
@@ -118,7 +118,7 @@ You can define the parent filter as an [AttributeFilter](30_tips__filter_visual_
 | filter | false | [Filter](30_tips__filter_visual_components.md) | The attribute filter definition |
 | parentFilters | false | AttributeFiltersOrPlaceholders[] | An array of parent attribute filter definitions |
 | connectToPlaceholder | false | IPlaceholder<IAttributeFilter> | The [visualization definition placeholder](30_tips__placeholders.md) used to get and set the value of the attribute filter |
-| parentFilterOverAttribute | false | ObjRef | The reference to the parent filter attribute over which the available options are reduced |
+| parentFilterOverAttribute | false | ObjRef &#124; Function | The reference to the parent filter attribute over which the available options are reduced, or a function called for every parent filter that returns such reference for the given parent filter |
```
Comment (evgeniabogranskaya) [inline docs/10_vis__attribute_filter_button_component.md:121]: The reference to the parent filter attribute over which the available options are reduced, or the function called for every parent filter that returns such reference for the given parent filter

File: docs/10_vis__attribute_filter_component.md [hunk 2]
```diff
@@ -118,7 +118,7 @@ You can define the parent filter as an [AttributeFilter](30_tips__filter_visual_
 | filter | false | [Filter](30_tips__filter_visual_components.md) | The attribute filter definition |
 | parentFilters | false | AttributeFiltersOrPlaceholders[] | An array of parent attribute filter definitions |
 | connectToPlaceholder | false | IPlaceholder<IAttributeFilter> | The [visualization definition placeholder](30_tips__placeholders.md) used to get and set the value of the attribute filter |
-| parentFilterOverAttribute | false | ObjRef | The reference to the parent filter attribute over which the available options are reduced |
+| parentFilterOverAttribute | false | ObjRef &#124; Function | The reference to the parent filter attribute over which the available options are reduced, or a function called for every parent filter that returns such reference for the given parent filter |
```
Comment (evgeniabogranskaya) [inline docs/10_vis__attribute_filter_component.md:121]: The reference to the parent filter attribute over which the available options are reduced, or the function called for every parent filter that returns such reference for the given parent filter

**Review by plavemar — APPROVED** [review]
LGTM

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-ui-documentation-validation/846/ : SUCCESS in 58s
- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-oss-check/555/ : SUCCESS in 6s

**Comment by gdgate** [comment]
Build succeeded (gate pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-ui-documentation-validation/847/ : SUCCESS in 1m 00s
- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-fossa-scanning-zuul-el7/153/ : SUCCESS in 51s (non-voting)
- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sonarqube-scan-zuul-gate/51/ : SUCCESS in 37s
- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-merge-compliance-check/241/ : SUCCESS in 16s

**Pull request merged**